airflow/api_connexion/openapi/v1.yaml

```diff
@@ -377,6 +377,13 @@ paths:
       parameters:
         - $ref: '#/components/parameters/PageLimit'
         - $ref: '#/components/parameters/PageOffset'
+        - name: only_active
+          in: query
+          schema:
+            type: boolean
+            default: true
```

Comment:
Is it breaking change, i think we should add note about it. I personally would have set it on False and only in one of the next versions would change the value to True.

Reply:
I agree - though setting this to True by default was Ash's suggestion - @ashb, is there a reason that we should be setting this to true by default or should we default to non-breaking behaviour?

Reply:
This was a bug that we included non-active dags in the API results in first place -- non-active dags can't be operated on (they likely don't exist on the filesystem anymore), and they are essentially invisible to the rest of the system.

My view was that most people wouldn't notice us changing this to true by default -- for instance if we changed the query but didn't add a param to control it would that be a breaking change?

I don't feel strongly about the default if you both think it should be False.

Reply:
https://apache-airflow.slack.com/archives/CCQ7EGB1P/p1614076697109600?thread_ts=1611774898.219100&cid=CCQ7EGB1P -- A User thought showing a deleted dag was a bug in the API.

Having the default as `True` gives us the same result as we have on the Webserver -- which would be ideal.

Reply:
Alright - thanks for the explanation! I'll leave it as is, in which case I think this PR is good to go.